src/pkg/scan/export/export_data_selector.go

```diff
@@ -1,7 +1,6 @@
 package export
 
 import "github.com/bmatcuk/doublestar"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380